changeset.go

```diff
@@ -97,7 +101,7 @@ type Changeset struct {
 }
 
 // Upsert upserts resource in a context of a changeset
-func (cs *Changeset) Upsert(ctx context.Context, changesetNamespace, changesetName, id string, data []byte) error {
+func (cs *Changeset) Upsert(ctx context.Context, data []byte, force bool) error {
```

Comment:
Yeah. Probably we need to split the command into an update and upsert. For the update force will make sense.
The point of adding force was to make an ability to update the resource even if it was already updated. Otherwise, if you call upsert for the second time you intentionally or non-intentionally rewrites the old one(original) resource, which makes revert impossible.